build.sbt

```diff
@@ -45,8 +43,8 @@ lazy val root = (project in file("."))
     PlayKeys.playDefaultPort := 9391,
     majorVersion := 0,
     scalaVersion := currentScalaVersion,
-    scalacOptions += "-Wconf:cat=deprecation:warning-verbose",
-    scalacOptions += "-Wconf:cat=unused-imports&src=routes/.*:s",
+    scalacOptions += "-Wconf:msg=unused import&src=html/.*:s",
+    scalacOptions += "-Wconf:msg=Flag.*repeatedly:s",
     scalacOptions += "-Wconf:src=routes/.*:s",
     resolvers ++= Seq(
```

Comment:
add
     ` Test / unmanagedSourceDirectories += baseDirectory.value / "it" / "test",`

After this and the change in scaladmt.conf do
  `sbt scalafmtCheckAll`